dts/bindings/ethernet/microchip,ksz8081.yaml

```diff
@@ -8,15 +8,13 @@ compatible: "microchip,ksz8081"
 include: ethernet-phy.yaml
 
 properties:
-  mc,reset-gpio:
+  reset-gpios:
     type: phandle-array
-    specifier-space: gpio
     description: GPIO connected to PHY reset signal pin. Reset is active low.
-  mc,interrupt-gpio:
+  interrupt-gpios:
```

Comment:
Looking at `dts/bindings` folder, `int-gpios` name is used often, `interrupt-gpios` only once.

Reply:
true, I'll change it to this